LunaMary/src/Util.java

```diff
@@ -0,0 +1,25 @@
+public class Util {
+
+
+    public Util() {
+
+    }
+
+    public Boolean validateSexo(String data) {
+        boolean textValid = false;
+
+        if (!data.isEmpty() && data.trim().length() == 1 && textValid == false) {
+
+            textValid = true;
+        }
+
+        return textValid;
+    }
+
+     public static char generateSigla(Integer num) {
```

Comment:
Interesting method, I actually like this solution. Good work.
Now you should only change its name to English